maestro/audit.py

```diff
@@ -56,6 +57,12 @@ def _format_what_compact(self, what):
         return '{} container{}'.format(
             len(what), 's' if len(what) > 1 else '')
 
+    def _format_what_services_ships(self, what):
+        if isinstance(what, entities.Entity):
+            return None
+        return(','.join(map(lambda e: '{}_{}'.format(e.service.name,
```

Comment:
That's... oddly specific. Isn't there a better way to pass that information to the script being executed? Would it make sense to build a JSON document and pass it via stdin? In any case, it would have to be documented.